app/assets/javascripts/i18n.js

```diff
@@ -497,6 +510,63 @@
     return message.replace(/_#\$#_/g, "$");
   };
 
+  // Not part of the public API
+  I18n.splitInterpolate = function(message, matches, options) {
+    var value
+    , placeholder
+    , name
+    , regex;
+
+    var buffer = [message];
```

Comment:
This is wrapping the original message to an array with one element.

If the `message` is e.g. "My name is {{name}} and I'm {{age}} years old" then:

1. `buffer = ["My name is {{name}} and I'm {{age}} years old"]`
2. After first `while` loop iteration: `buffer = ["My name is ", "Mikko", " and I'm {{age}} years old"]`
3. After second iteration: `buffer = ["My name is ", "Mikko", " and I'm ", 29, " years old"]`